source/includes/stackpath/_workloads.md

```diff
@@ -0,0 +1,115 @@
+## Workloads
+
+Deploy and manage your workloads.
+
```

Comment:
It would be nice to specify what a workload is. Similarly to what is done in the Azure plugin doc under networking. Just a suggestion. :)

Reply:
f7751db